src/hotspot/share/oops/method.cpp

```diff
@@ -867,6 +867,10 @@ bool Method::needs_clinit_barrier() const {
   return is_static() && !method_holder()->is_initialized();
 }
 
+bool Method::is_object_wait0() const {
```

Comment:
It might be worth mentioning that is not a general-purpose API, so we don't have to worry about false positives here.